src/main/resources/feedbackQuestionRubricEditFormWeightFragment.html

```diff
@@ -1,3 +1,3 @@
 <th class="rubricCol-${questionIndex}-${col}">
-  <input type="number" class="form-control nonDestructive" value="${rubricWeight}" id="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT}-${questionIndex}-${col}" name="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT}-${col}" step="0.01">
+  <input type="number" class="form-control nonDestructive" value="${rubricWeight}" id="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT}-${questionIndex}-${col}" name="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT}-${col}" step="0.01" required>
```

Comment:
The input should also not be `required` by default (as the rubric weight is not enabled by default).

Reply:
Okay.